node/core/approval-voting/src/backend.rs

```diff
@@ -52,6 +54,11 @@ pub trait Backend {
 		&self,
 		candidate_hash: &CandidateHash,
 	) -> SubsystemResult<Option<CandidateEntry>>;
+	/// Load a candidate entry from the DB with scheme version 1.
+	fn load_candidate_entry_v1(
```

Comment:
Style: given that this isn't needed for normal operation of the backend, I would recommend a 

```rust
trait V1ReadBackend: Backend { ... }
```

which contains this function.